acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/signer/SignerConfigurationBuilder.java

```diff
@@ -69,11 +70,20 @@ public SignerConfigurationBuilder withServerTlsOptions(final TlsOptions serverTl
     return this;
   }
 
-  public void withOverriddenCA(final TlsCertificateDefinition keystore) {
+  public SignerConfigurationBuilder withOverriddenCA(final TlsCertificateDefinition keystore) {
     this.overriddenCaTrustStore = keystore;
+    return this;
+  }
+
+  public SignerConfigurationBuilder withMode(final String mode) {
```

Comment:
nit: just an idea .... what about withFilecoinMode(), withEth1Mode(), withEth2Mode() methods?

Reply:
actually adds additional complexity when trying to switch between modes (based on keytype).